build.gradle.kts

```diff
@@ -24,11 +24,12 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+@file:Suppress("RemoveRedundantQualifierName")
```

Comment:
Please don't suppress warnings for such a big scope. We don't want to skip the cases in which the qualifier name is really redundant.